server/asset/btc/btc.go

```diff
@@ -194,9 +214,30 @@ type BackendCloneConfig struct {
 	Net            dex.Network
 	ChainParams    *chaincfg.Params
 	Ports          dexbtc.NetPorts
-	// FeeEstimator provides a way to get fees given an RawRequest-enabled
-	// client and a confirmation target.
-	FeeEstimator func(*RPCClient) (uint64, error)
+	// ManualFeeScan specifies that median block fees should be calculated by
+	// scanning transactions since the getblockstats rpc is not available.
+	// Median block fees are used to estimate fee rates when the cache is not
+	// primed.
+	ManualMedianFee bool
+	// NoCompetitionFeeRate specifies a fee rate to use if estimatesmartfee
+	// or estimatefee aren't ready and the median fee is finding relatively
+	// empty blocks.
+	NoCompetitionFeeRate uint64
+	// DumbFeeEstimates is for asset's whose RPC is estimatefee instead of
+	// estimatesmartfee.
+	DumbFeeEstimates bool
+	// Argsless fee estimates are for assets who don't take an argument for
+	// number of blocks to estimatefee.
+	ArglessFeeEstimates bool
+	// FeeConfs specifies the target number of confirmations to use for
+	// estimate(smart)fee. If not set, default value is 1,
+	FeeConfs int64
+	// MaxFeeBlocks is the maximum number of blocks that can be evaluated for
+	// median fee calculations. If > 100 txs are not seen in the last
+	// MaxFeeBlocks, an ErrNoCompetition is returned. Default value is 5.
```

Comment:
Makes sense, but the `ErrNoCompetition` seems to be internal, not really something of concern for consumer of `BackendCloneConfig`